src/Modal/_ModalDialog.scss

```diff
@@ -285,5 +285,36 @@
 
   .pgn__modal-title {
     font-size: $h4-font-size;
+    display: flex;
+    flex-grow: 1;
+    align-items: center;
+
+    & > * {
+      margin: 0;
+    }
```

Comment:
[curious] Is this style property necessary? Removing it doesn't seem to change the resulting visual styles in the browser.